index.html

```diff
@@ -81,11 +81,105 @@ <h1>Skills</h1>
+                            <li class="border-right">css</li>
+                            <li class="border-right">html</li>
+                            <li class="border-right">bootstrap</li>
+                            <li>Ruby</li>
+                        </ul> 
+                        <button type="button" class="myprojects">See projects</button>
+                    </div>
+                    
+
+                    <div class="projects-container">
+                        <div class="image-placeholder">
+                            <img src="IMG1/placeholder.png" alt="">
+                        </div>
+                        <div class="content">
+                            <div class="background-img2">
+                            </div>
+                                <div class="text-area">
+                                    <h1>Multi-Post Stories</h1>
+                                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                                </div>
+                                <ul class="list-tags">
+                                    <li class="border-right">css</li>
+                                    <li class="border-right">html</li>
+                                    <li class="border-right">bootstrap</li>
+                                    <li>Ruby</li>
+                                </ul>  
+                            <button type="button" class="myprojects">See projects</button>
+                        </div>
+
+                        <div class="projects-container">
+                            <div class="image-placeholder">
+                                <img src="IMG1/placeholder.png" alt="">
+                            </div>
+                            <div class="content">
+                                <div class="background-img3">
+                                </div>
+                                    <div class="text-area">
+                                        <h1>Multi-Post Stories</h1>
+                                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                                    </div>
+                                    <ul class="list-tags">
+                                        <li class="border-right">css</li>
+                                        <li class="border-right">html</li>
+                                        <li class="border-right">bootstrap</li>
+                                        <li>Ruby</li>
+                                    </ul> 
+                                <button type="button" class="myprojects">See projects</button>
```

Comment:
- [ ]  Kindly follow the button styling: Active, Hover and Pressed check screenshot below ⬇️ 
![btn](https://user-images.githubusercontent.com/69638013/154810789-aa11b66a-1769-4a4d-bab3-d2d3ef82e5bc.PNG)